src/js/BookReader.js

```diff
@@ -424,7 +450,9 @@ BookReader.prototype.init = function() {
   this.trigger(BookReader.eventNames.PostInit);
 
   // Return to default
-  this.suppressFragmentChange = false;
+  if (!this.options.initialSearchTerm) {
+    this.suppressFragmentChange = false;
```

Comment:
Q: what does the comment `Return to default` mean w/r/t `suppressFragmentChange`?

Reply:
improved comment:
`// If not searching, set to allow on-going fragment changes`